src/configs/recommended.ts

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+// tslint:disable object-literal-contextual-type
```

Comment:
API break!! not down for this.

Reply:
This is a comment, how does it break the API?